examples/plotting/200_cell_transitions.ipynb

```diff
@@ -101,38 +101,11 @@
      "name": "stdout",
```

Comment:
<p>.... This gives the likelihood of each single cell being mapped to a specific cluster.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>